CleanWeather/Scenes/FavouriteCities/FavouriteCitiesPresenter.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  FavouriteCitiesPresenter.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 05/12/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+protocol FavouriteCitiesPresenter: SpinnerPresenter, AlertPresenter {
+    func presentCities(city: [City], favourites: [FavouriteCity])
+}
+
+final class FavouriteCitiesPresenterImpl<T: FavouriteCitiesPresentable>: SharedPresenter<T> {
+
+    private var displayable = [FavouriteCitiesListDisplayable]()
+}
+
+extension FavouriteCitiesPresenterImpl: FavouriteCitiesPresenter {
+
+    func presentCities(city: [City], favourites: [FavouriteCity]) {
+
+        //FIXME: Is it possible to do it with map/filter?
```

Comment:
wiele razy probowalem zrobic map/filter w map/filter i chyba nigdy nie mialem na tyle czasu zeby zagłębić temat, zwyklem for eachem tez jest ok